CHANGELOG.md

```diff
@@ -102,15 +102,15 @@ Color=#ccffcc
 ### Changed
 - automatic update GPO handling, now much more user friendly
 - reworked error handling to allow limited non admin operation
-- reworked status codes for better ui expirience
-- when download fails but the file was already downloaded in the previuse session the old file is used
+- reworked status codes for better ui experience
+- when download fails but the file was already downloaded in the previous session the old file is used
 - reworked UAC bypass handling
 
 ### Fixed
 - windows 10 version detection
-- issue when started rom a read only directory, fallback to ...\{UserProfile}\Downloads\WuMgr\
+- issue when started from a read only directory, fallback to ...\{UserProfile}\Downloads\WuMgr\

```

Comment:
* `read only` should be `read-only`
* `fallback` should be `fall back`
* `...\{UserProfile}` should probably just say `%USERPROFILE%` (optional change, your call)

Reply:
I don't think it should be `fall back`

Reply:
Incorrect: "fallback" is a noun, "fall back" is a verb + adverb (i.e. to fall back on/upon).

I think what you're doing is looking for random GitHub repositories to contribute something to, to add activity to your GH profile.  You are not improving this PR.  Please knock it off.